src/implementations/twig/components/blockquote/blockquote.html.twig

```diff
@@ -39,8 +44,23 @@
 {# Print the result #}
 
 <blockquote class="{{ _css_class }}"{{ _extra_attributes|raw }}>
```

Comment:
This is a bit hard to implement, as i need an inner container that should those borders left and on responsive the border should change to the outer container, also i'm not sure about a figure tag containing all the markup. Maybe i did not understood corectly

Reply:
what i'm saying is just that instead of wrapping everything into a <blockquote> tag this tagcould be only used to wrap the quote, i see that we need some markup to handle the look and feel, the rest of the proposal is indeed only about what selector should identify what, not changing the current wrappers you have.
So,  the ` __body` element could be what currently is the `__inner` element, and the `<blockquote>` could be identified by a new element `__quote`.
The resulting structure would be:
```
<figure class="ecl-blockquote>
  <div class="ecl-blockquote__body>
    <blockquote class="ecl-blockquote__quote"><p></p></blockquote>
    <footer class="ecl-blockquote__attribution></footer>
  </div>
  <img class="ecl-blockquote__image />
</figure>
```

Reply:
Should be done.